sanity-io/structured-content-2022
# New black color and updated footer

# New black color and updated footer styles

## Intent

Solves this Shortcut Story.

## Testing this PR

1. Visit the [Preview Deploy for this PR](https://structured-content-2022-web-git-new-black-color-and-upda-5c89af.sanity.build/)
2. Verify that the new footer renders in accordance with the [latest Figma sketches](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=53%3A410)
